config/x_hh_bbtautau_run2.yaml

```diff
@@ -24,16 +28,1318 @@ processes:
+  - name: CMS_eff_t_id_etauFR_endcaps_2016
+    type: auto
+    eras:
+    - Run_2016
+  - name: CMS_eff_t_id_etauFR_endcaps_2016_HIPM
+    type: auto
+    eras:
+    - Run_2016_HIPM
+  - name: CMS_eff_t_id_mutauFR_eta0p4to0p8_2018
+    type: auto
+    eras:
+    - Run_2018
+  - name: CMS_eff_t_id_mutauFR_eta0p4to0p8_2017
+    type: auto
+    eras:
+    - Run_2017
+  - name: CMS_eff_t_id_mutauFR_eta0p4to0p8_2016
+    type: auto
+    eras:
+    - Run_2016
+  - name: CMS_eff_t_id_mutauFR_eta0p4to0p8_2016_HIPM
+    type: auto
+    eras:
+    - Run_2016_HIPM
+  - name: CMS_eff_t_id_mutauFR_eta0p8to1p2_2018
+    type: auto
+    eras:
+    - Run_2018
+  - name: CMS_eff_t_id_mutauFR_eta0p8to1p2_2017
+    type: auto
+    eras:
+    - Run_2017
+  - name: CMS_eff_t_id_mutauFR_eta0p8to1p2_2016
+    type: auto
+    eras:
+    - Run_2016
+  - name: CMS_eff_t_id_mutauFR_eta0p8to1p2_2016_HIPM
+    type: auto
+    eras:
+    - Run_2016_HIPM
+  - name: CMS_eff_t_id_mutauFR_eta1p2to1p7_2018
+    type: auto
+    eras:
+    - Run_2018
+  - name: CMS_eff_t_id_mutauFR_eta1p2to1p7_2017
+    type: auto
+    eras:
+    - Run_2017
+  - name: CMS_eff_t_id_mutauFR_eta1p2to1p7_2016
+    type: auto
+    eras:
+    - Run_2016
+  - name: CMS_eff_t_id_mutauFR_eta1p2to1p7_2016_HIPM
+    type: auto
+    eras:
+    - Run_2016_HIPM
+  - name: CMS_eff_t_id_mutauFR_etaGt1p7_2018
+    type: auto
+    eras:
+    - Run_2018
+  - name: CMS_eff_t_id_mutauFR_etaGt1p7_2017
+    type: auto
+    eras:
+    - Run_2017
+  - name: CMS_eff_t_id_mutauFR_etaGt1p7_2016
+    type: auto
+    eras:
+    - Run_2016
+  - name: CMS_eff_t_id_mutauFR_etaGt1p7_2016_HIPM
+    type: auto
+    eras:
+    - Run_2016_HIPM
+  - name: CMS_eff_t_id_mutauFR_etaLt0p4_2018
+    type: auto
+    eras:
+    - Run_2018
+  - name: CMS_eff_t_id_mutauFR_etaLt0p4_2017
+    type: auto
+    eras:
+    - Run_2017
+  - name: CMS_eff_t_id_mutauFR_etaLt0p4_2016
+    type: auto
+    eras:
+    - Run_2016
+  - name: CMS_eff_t_id_mutauFR_etaLt0p4_2016_HIPM
+    type: auto
+    eras:
+    - Run_2016_HIPM
+  #- name: CMS_eff_t_id_stat_highpT_bin1_2018
+  #  type: auto
+  #  eras:
+  #  - Run_2018
+  #- name: CMS_eff_t_id_stat_highpT_bin1_2017
+  #  type: auto
+  #  eras:
+  #  - Run_2017
+  #- name: CMS_eff_t_id_stat_highpT_bin1_2016
+  #  type: auto
+  #  eras:
+  #  - Run_2016
+  #- name: CMS_eff_t_id_stat_highpT_bin1_2016_HIPM
+  #  type: auto
+  #  eras:
+  #  - Run_2016_HIPM
+  #- name: CMS_eff_t_id_stat_highpT_bin2_2018
+  #  type: auto
+  #  eras:
+  #  - Run_2018
+  #- name: CMS_eff_t_id_stat_highpT_bin2_2017
+  #  type: auto
+  #  eras:
+  #  - Run_2017
+  #- name: CMS_eff_t_id_stat_highpT_bin2_2016
+  #  type: auto
+  #  eras:
+  #  - Run_2016
+  #- name: CMS_eff_t_id_stat_highpT_bin2_2016_HIPM
+  #  type: auto
+  #  eras:
+  #  - Run_2016_HIPM
+  #- name: CMS_eff_t_id_syst_highpT_bin1
+  #  type: auto
+  #- name: CMS_eff_t_id_syst_highpT_bin2
+  #  type: auto
+  #- name: CMS_eff_t_id_syst_highpT_extrap
+  #  type: auto
```

Comment:
why these uncertainteis are commented?

Reply:
because in an earlier production they were not included. Uncommenting them rn